README.markdown

```diff
@@ -533,21 +533,50 @@ UIView.animate(withDuration: 1.0, animations: {
 }) { f in
   self.myView.removeFromSuperview()
 }
+
 ```
 For single-expression closures one one line, use implicit returns:
 
 ```swift
 attendeeList.sort { $0 > $1 }
 ```
 
-For single-expression closures with more than one, use explicit returns:
+For single-expression closures with more than one line, use explicit returns:
 
 ```swift
-attendeeList.sort { a, b in 
+attendeeList.map { a, b in 
+  let sum = a + b
+  return sum
+}
+```
+
+For closures with more than one line, use named parameters
+**Preferred**:
+
+```swift
+attendeeList.sort { a, b in
   return a > b
 }
 ```
 
+**Not Preferred**:
+```swift
+attendeeList.sort {
+  return $0 > $1
+}
+```
+
+If possible avoid more than one line if the expression is easy to evaluate
+**Preferred**:
+```swift
+attendeeList.sort { $0 > $1 }
+```
+**Not Preferred**:
+```swift
+attendeeList.sort { a,b in
+  return a > b
+}
+```
```

Comment:
I think the amount of statements is a better/more strict metric. How about "For closures with more than one statement"?

Reply:
@alexcristici are you okay with this?

Reply:
@alexcristici still waiting for your thoughts on this one

Reply:
Agree, sorry didn’t see it.